packages/arch3-proto/package.json

```diff
@@ -60,6 +60,7 @@
     "test:debug": "node --inspect node_modules/.bin/jest --runInBand"
   },
   "dependencies": {
+    "@archwayhq/arch3-core": "0.1.0",
```

Comment:
This will create a circular reference. `arch3-proto` should be used inside `arch3-core`, not the other way around.